packages/ecs-agent/ecs-agent.spec

```diff
@@ -1,31 +1,43 @@
-%global goproject github.com/aws
-%global gorepo amazon-ecs-agent
-%global goimport %{goproject}/%{gorepo}
+%global agent_goproject github.com/aws
+%global agent_gorepo amazon-ecs-agent
+%global agent_goimport %{agent_goproject}/%{agent_gorepo}
 
-%global gover 1.43.0
+%global agent_gover 1.43.0
 # git rev-parse --short=8
-%global gitrev 1ebf0604
+%global agent_gitrev 1ebf0604
+
+%global ecscni_goproject github.com/aws
+%global ecscni_gorepo amazon-ecs-cni-plugins
+%global ecscni_goimport %{ecscni_goproject}/%{ecscni_gorepo}
+%global ecscni_gitrev 9066095fbab8ca7b659ab2b95a1590e7aee55545
 
 # Construct reproducible tar archives
 # See https://reproducible-builds.org/docs/archives/
 %global source_date_epoch 1234567890
 %global tar_cf tar --sort=name --mtime="@%{source_date_epoch}" --owner=0 --group=0 --numeric-owner -cf
 
 Name: %{_cross_os}ecs-agent
-Version: %{gover}
+Version: %{agent_gover}
 Release: 1%{?dist}
 Summary: Amazon Elastic Container Service agent
 License: Apache-2.0
-URL: https://%{goimport}
-Source0: https://%{goimport}/archive/v%{gover}/amazon-ecs-agent-v%{gover}.tar.gz
-Source1: ecs.service
-Source2: ecs-tmpfiles.conf
-Source3: ecs-sysctl.conf
-Source4: ecs.config
-Source5: pause-image-VERSION
-Source6: pause-config.json
-Source7: pause-manifest.json
-Source8: pause-repositories
+URL: https://%{agent_goimport}
+Source0: https://%{agent_goimport}/archive/v%{agent_gover}/%{agent_gorepo}-v%{agent_gover}.tar.gz
+Source1: https://%{ecscni_goimport}/archive/%{ecscni_gitrev}/%{ecscni_gorepo}.tar.gz
+Source101: ecs.service
+Source102: ecs-tmpfiles.conf
+Source103: ecs-sysctl.conf
+Source104: ecs.config
+Source105: pause-image-VERSION
+Source106: pause-config.json
+Source107: pause-manifest.json
+Source108: pause-repositories
+
+# Patches are numbered according to which source they apply to
+# Patches 0000 - 0999 apply to Source0
+# Patches 1000 - 1999 apply to Source1
+# Patches 2000 - 2999 apply to Source2
```

Comment:
this comment nicely describes how patches are numbered, however `Source2` does not exist, would it confuse the reader ?

Reply:
`Source2` will be added before this PR is ready to merge.

Reply:
The latest push now includes `Source2`.